roles/openshift_health_checker/openshift_checks/package_availability.py

```diff
@@ -1,9 +1,9 @@
 # pylint: disable=missing-docstring
 from openshift_checks import OpenShiftCheck, get_var
-from openshift_checks.mixins import NotContainerizedMixin
+from openshift_checks.mixins import NotContainerizedMixin, NeedsYumMixin
 
 
-class PackageAvailability(NotContainerizedMixin, OpenShiftCheck):
+class PackageAvailability(NotContainerizedMixin, NeedsYumMixin, OpenShiftCheck):
```

Comment:
This is about having the Python package `yum` available/installed. Not that we couldn't support DNF, just that we don't have code for it at the moment.

This PR makes the check be skipped instead of giving an spurious error.